ocfweb/static/scss/pages/account/_register.scss

```diff
@@ -3,3 +3,18 @@
         text-align: center;
     }
 }
+
+.recommendation {
+    padding: 8px;
+}
+
+.recommendations-list {
+    margin: 7px;
+    overflow-y: scroll;
+    max-height: 150px;
+}
+
+.username-feedback {
+    display: none;
+    padding: 7px;
+}
```

Comment:
this stuff all needs to go in the `.page-register` block above, otherwise it's still part of the global css

(we only ship one css file currently, the way we namespace is based on the `.page-blah` rules